src/components/sound.js

```diff
@@ -57,16 +59,19 @@ module.exports.Component = registerComponent('sound', {
     if (data.on !== oldData.on) {
       this.updateEventListener(oldData.on);
     }
-
     // All sound values set. Load in `src`.
     if (srcChanged) {
       var self = this;
       this.audioLoader.load(data.src, function (buffer) {
         self.pool.children.forEach(function (sound) {
           sound.setBuffer(buffer);
         });
+        self.loaded = true;
+
         // Remove this key from cache, otherwise we can't play it again
         THREE.Cache.remove(data.src);
+
+        if (self.data.autoplay || self.mustPlay) { self.playSound(); }
```

Comment:
For me `isSoundPlaying` feels like the sound is already playing not that we've asked the sound to start playing and it was delayed until the buffer is loaded